programs/cross-chain-escrow-dst/tests/test_cross_chain_escrow_dst_native.rs

```diff
@@ -261,6 +263,47 @@ mod test_escrow_native {
             .is_none());
     }
 
+    #[test_context(TestState)]
+    #[tokio::test]
+    async fn test_public_withdraw_fail_with_unwhitelisted_resolver(test_state: &mut TestState) {
```

Comment:
It seems there’s no need to have this test in the _native section, and it would be better to move it to the SPL tests, alongside the other similar ones